atomic_cms.gemspec

```diff
@@ -4,10 +4,10 @@ Gem::Specification.new do |s|
   s.summary     = 'Atomic CMS'
   s.description = 'Live CMS powered by atomic assets.'
   s.authors     = ['Don Humphreys', 'Spartan']
```

Comment:
I dont think so. Is it incorrect though that Don Humphreys @ Spartan is / was the author?

Thinking we just change it to `'Very'`, here?